executor/dispatch/dispatch.go

```diff
@@ -0,0 +1,27 @@
+package executors
```

Comment:
WDYT about calling this 'executordispatch'?  Package names that have no relationship to their import names cause a huge jump the mental overhead of reading for me.

Joining the last two packages is the first compromise that comes to mind that gives the package name and import name some relationship, and also addresses the irritation of three 'dispatch' packages that would force us to alias them on every import.  Open to other ideas, that's just my first.